saleor/setting/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.utils.translation import pgettext_lazy
+from django.utils.encoding import python_2_unicode_compatible
+
+
+INTEGER = 'i'
+STRING = 's'
+BOOLEAN = 'b'
+
+
+@python_2_unicode_compatible
+class Setting(models.Model):
+    VALUE_TYPE_CHOICES = (
+        (INTEGER, pgettext_lazy('Settings', 'Integer')),
+        (STRING, pgettext_lazy('Settings', 'String')),
+        (BOOLEAN, pgettext_lazy('Settings', 'Boolean')),
+    )
+    name = models.CharField(
+        pgettext_lazy('Settings field', 'name'), max_length=128)
+    value_type = models.CharField(pgettext_lazy('Settings field', 'value type'),
```

Comment:
I think we should only cover the string → string mapping. Value type depends on the setting name, it doesn't make sense to pair the same setting with multiple arbitrary types.